### PR TITLE
HW01 - Maria Benavides

### PR DESCRIPTION
**Pull request by mabenavides**
Hi, 

I edit the README to add some professional and personal information. It was challenging. It was the first time I used Markdown. 

I had trouble with linking GitHub and RStudio. I followed the instructions, but at first, no Git tab appeared. I tried closing all sessions and it suddenly appeared. 

I'm not sure how "cloning" works. I thought the repository was supposed to appear also on my computer (on the folder I selected) but this is not happening. 

Finally, I also had trouble with adding the image. I asked someone who's very skilled with R, and that was very helpful, however, I don't know if I followed the expected process for adding an image (though it worked).

### CODE REVIEW

**Review by yirouf — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |+           |              |            |
| **Coding strategy**         |+           |              |            |
| **Presentation**            |+           |              |            |
| **Achievement, creativity** |+           |              |            |
| **Reproducibility**         |+           |              |            |

Remarks:

*Firstly, you have completed all the requirements!
* I really enjoyed reading your biography! I think you incorporated list, bold, link and image embedding very well. Everything is well reflected in your workflow, and I think the way you used to specify actions in the commit is very efficient. 
* Please say hi to Pepper for me (maybe through skype)! It's good to see a little friend during this difficult time. I also have an interest in developmental psychology and education (parental aspects). So very nice meeting you!
* I was having difficulty embedding the image. Now I see how to do it potentially! 
* For your question about cloning, I think this site may be able to clarify something for you! (https://help.github.com/en/desktop/contributing-to-projects/cloning-and-forking-repositories-from-github-desktop)

**Review by Eunsil-Choi — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     o      |              |            |
| **Coding strategy**         |    o       |              |            |
| **Presentation**            |     o      |              |            |
| **Achievement, creativity** |     o      |              |            |
| **Reproducibility**         |     o      |              |            |

Hello, Maria! I like your bio and think you did a wonderful job.

- You successfully managed to follow the assignment instructions including header, emphasis(italic, bold), and list.
-  I also like the way you introduce yourself with a balanced professional and personal information; especially I enjoyed reading your professional skills and interests which I also believe are greatly significant public policy areas. 
- Things that you need additional work are inserting a link and image. In the line of code in you Markdown, I found there is no problem at all, but I cannot see the image, and the link got lost when I tried to open. I had similar problems with an image, but I just changed the image file from local directory to the one from online resources. If I find a way to fix it, I will definitely share with you. 

Overall, you did a great job, and congrats on your successful first try in using Markdown!

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+## Maria Benavides Bio
```
Comment (nearridge) [inline README.md:2]: You want an empty line between headers FYI. Sometimes not having one will cause rendering problems (not in this case however).

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+***
```
Comment (nearridge) [inline README.md:3]: nice insertion of a neutral line across the page.

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
...
+I was born in Bogota, Colombia. I consider myself a policy professional with international experience in social development, specifically in **childhood development and education**. I am skilled at decomposing, analyzing, and evaluating policy issues to devise possible solutions; for instance, I built new quality measurement tools for early childhood education services in Colombia, which the Ministry of National Education later piloted and adopted in 300 municipalities.
```
Comment (nearridge) [inline README.md:4]: Good use of emphasis through bolding and a nice description of your background.

File: README.md [hunk 4]
```diff
@@ -1 +1,17 @@
...
+
+I have cultivated a new interest in the intersection between conflict and childhood development; looking for professional opportunities that allow me to apply her social policy expertise within violent and conflict contexts. Here you can find my [LinkedIn profile](www.linkedin.com/in/maria-benavides-c)
```
Comment (nearridge) [inline README.md:6]: Great job with this link

File: README.md [hunk 5]
```diff
@@ -1 +1,17 @@
...
+
+Now, on a more *personal level*, I am the younger of three, I have two brothers. I miss my familiy very much, especially under the current circumstances. I love dogs. Back in Colombia I had the cutest mix breed dog, her name is Pepper. My parents are taking care of her at the moment. Here is a picture of her lovely sweet face. 
+
+![Pepper, the cutest dog](Pepper_HW1.jpeg)
+
+During this quarantine time, I have been trying to adjust by having a routine. Some of the activities I try to do daily are:
```
Comment (nearridge) [inline README.md:12]: This list looks great. I'm awful at smash.

File: README.md [hunk 6]
```diff
@@ -1 +1,17 @@
...
+# hw01
+## Maria Benavides Bio
+***
+I was born in Bogota, Colombia. I consider myself a policy professional with international experience in social development, specifically in **childhood development and education**. I am skilled at decomposing, analyzing, and evaluating policy issues to devise possible solutions; for instance, I built new quality measurement tools for early childhood education services in Colombia, which the Ministry of National Education later piloted and adopted in 300 municipalities.
+
+I have cultivated a new interest in the intersection between conflict and childhood development; looking for professional opportunities that allow me to apply her social policy expertise within violent and conflict contexts. Here you can find my [LinkedIn profile](www.linkedin.com/in/maria-benavides-c)
+
+Now, on a more *personal level*, I am the younger of three, I have two brothers. I miss my familiy very much, especially under the current circumstances. I love dogs. Back in Colombia I had the cutest mix breed dog, her name is Pepper. My parents are taking care of her at the moment. Here is a picture of her lovely sweet face. 
+
+![Pepper, the cutest dog](Pepper_HW1.jpeg)
```
Comment (nearridge) [inline README.md:10]: Nice job committing and inserting this image!

**Review by nearridge — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |     x      |              |            |
| **Reproducibility**         |      x     |              |            |

Remarks: If you are working in RStudio server, you should only be able to clone the repository to the server so you won't actually see it on your physical computer. That's an unfortunate sacrifice we had to make because we can't trouble shoot technical issues as easily this quarter due to remote learning. To your question about if you uploaded your image properly, it looks like it all worked out. Git is definitely not an easy thing to get a hang of, but it looks like you're getting it.

**Pull request closed without merging**